code/modules/antagonists/eldritch_cult/transmutations/mind_transmutations.dm

```diff
@@ -0,0 +1,49 @@
+/datum/eldritch_transmutation/mind_knife
+	name = "Cerebral Blade"
+	required_atoms = list(/obj/item/kitchen/knife,/obj/item/book) 
+	result_atoms = list(/obj/item/gun/magic/hook/sickly_blade/mind)
+	required_shit_list = "A book and a knife."
+
+/datum/eldritch_transmutation/eldritch_eye
+	name = "Eldritch Eye"
+	required_atoms = list(/obj/item/organ/eyes,/obj/item/flashlight) 
+	result_atoms = list(/obj/item/clothing/glasses/hud/toggle/eldritch_eye)
+	required_shit_list = "A pair of eyes and a flashlight"
+
+/datum/eldritch_transmutation/final/mind_final
+	name = "Beyond All Knowldege Lies Despair"
+	required_atoms = list(/mob/living/carbon/human)
+	required_shit_list = "Three dead bodies."
+
+/datum/eldritch_transmutation/final/mind_final/on_finished_recipe(mob/living/user, list/atoms, loc)
+	var/mob/living/carbon/human/H = user
+	///do the mind shaker, shake that mind
+	var/atom/movable/gravity_lens/shockwave = new(get_turf(user))
+	H.physiology.brute_mod *= 0.75
+	H.physiology.burn_mod *= 0.75
+	H.physiology.stamina_mod = 0
+	H.physiology.stun_mod = 0
```

Comment:
why 0 stam and stun mod rather than stun immunity trait

Reply:
Took it from rust heretic, just kinda how they wrote it.

Reply:
Would you prefer if I changed it to the stun immune trait?

Reply:
So I did some testing on this vs the stun immune trait, and while the trait DOES make you stun immune, you still get slowed from stamina damage and also jittery as fuck, which I dont really think the ascended heretic needs to be affected by since they're supposed to be super strong (although they're not unkillable, I will look to see if this is needed in the future)